### ISSUE TITLE
Sony Vegas and openencodevfw

### ISSUE THREAD

**Issue by suppy1** [jackun/openencodevfw#7]
Hey guys.. Got a question hope you can help?

I've captured game footage with openencodevfw but when I try to open in Sony Vegas 12 it won't work? Sony Vegas refuses to open it.  Anyway to make this work in sony vegas without losing quality?

**Comment by acidburn974** [comment]
[Imgur](http://i.imgur.com/iwuFCza.png)

Here is my configuration using Dxtory, I record on 720p60fps with 2 audio channels in MP3-ACM. I have K-Lite Codec Pack installed on my PC. I hope this comment will help people.

**Comment by IBIubbleTea** [comment]
@EriksGit Just wondering but where do I put this line of code for me to hear both audio streams without having it extract it? 

ffmpeg -i input.avi -c:v copy -map 0 output.mp4

**Comment by jackun** [comment]
@IBIubbleTea You mean @EriksGit 's `Autoremux` helper? It should be already doing remuxing like that.

**Comment by IBIubbleTea** [comment]
@jackun I just recorded a test video and this is the out come.

Original video, Can only hear in game but not mic.
Converted video, Can only hear my mic and not in game 

Now I'm confused. Of course if I add it to Sony Vegas, I can hear both..

**Comment by jackun** [comment]
That might be you media player issue. It might think they are different language audio tracks etc so only plays one at a time.

**Comment by IBIubbleTea** [comment]
@jackun I'm using Windows Media Player for those tests. I just tried using VLC for the original and converted one and I can only hear in game for both, This is weird.

**Comment by jackun** [comment]
In VLC switch audio from menu Audio > Audio Track.

**Comment by IBIubbleTea** [comment]
Thanks for your help. I guess I will have to live with it. Until it's fixed or something.Like I used Largarith in the past and I have no problem with it, I can hear both audio tracks in windows media player..

**Comment by jackun** [comment]
If it worked then, it should work now too though. Maybe you have installed a custom AVI splitter so with that even WMP doesn't play audio tracks simultaneously.

**Comment by IBIubbleTea** [comment]
I mean Lagarith works and still does just not VCE

**Comment by jackun** [comment]
Yep, if you hear both audio tracks with lagarith, you should hear with VCE encoded h264 video too. You can try using MPC-HC and in `Play > Filter` menu (or right click) it shows what splitter/decoder filters it is using (and what WMP should be using too) and see what it does differently there.

**Comment by IBIubbleTea** [comment]
I don't know anymore. I can't seem to figure it out. I'll just leave it and if I want to edit something, I'll just open it up in Sony Vegas
